gnome-initial-setup/pages/endless-eula/gis-endless-eula-page.c

```diff
@@ -405,6 +406,14 @@ gis_endless_eula_page_constructed (GObject *object)
 
   gis_page_set_forward_text (GIS_PAGE (page), _("_Accept and Continue"));
   gis_page_set_complete (GIS_PAGE (page), TRUE);
+
+  /* Hide the page in demo mode; we still want to create it, since we
+   * want the save_data implementation to run at the end of the FBE.
+   * Also, make sure metrics are off in that case.
+   */
+  demo_mode = gis_driver_is_in_demo_mode (GIS_PAGE (page)->driver);
+  gtk_toggle_button_set_active (GTK_TOGGLE_BUTTON (priv->metrics_checkbutton), !demo_mode);
+  gtk_widget_set_visible (GTK_WIDGET (page), !demo_mode);
```

Comment:
Ah, yikes. I suppose that this is the best way to handle the problem for now, unless we want an "invisible but active" mode for pages in upstream GIS. We'll need to monitor if we end up doing something similar again.